app/_includes/md/kic/installation.md

```diff
@@ -0,0 +1,74 @@
+## Installation
+
+Please follow the [deployment](/kubernetes-ingress-controller/{{page.kong_version}}/deployment/overview) documentation to install
+the {{site.kic_product_name}} onto your Kubernetes cluster.
+
+If you wish to use the Gateway APIs examples, you must first [install the
+Gateway API resources](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api)
+and restart your Deployment after:
+
+{% navtabs codeblock %}
+{% navtab Command %}
+```bash
+kubectl rollout restart -n NAMESPACE deployment DEPLOYMENT_NAME
+```
+{% endnavtab %}
+{% navtab Response %}
+```text
+deployment.apps/DEPLOYMENT_NAME restarted
+```
+{% endnavtab %}
+{% endnavtabs %}
+
+Layer 4 routes (TCPRoute, UDPRoute, TLSRoute) are currently only available in
+the Gateway APIs experimental channel. To use them, install the experimental
+channel CRDs and enable the corresponding feature gate in
+{{site.kic_product_name}}:
+
+{% navtabs codeblock %}
+{% navtab Command %}
+```bash
+kubectl set env -n kong deployment/ingress-kong CONTROLLER_FEATURE_GATES="GatewayAlpha=true" -c ingress-controller
```

Comment:
This command works only for KIC 2.6 and later. WDYT about KIC versions <= 2.5?

Reply:
IMO gateway APIs are still beta because currently upstream are still using 0.x versions and `v1beat1` API version, and no implementation announces GA. I think it is OK to have the dual-API guide in 2.6 and later, because in 2.6 we have enabled gateway APIs by default.

Reply:
Latest version adds https://deploy-preview-4766--kongdocs.netlify.app/kubernetes-ingress-controller/2.7.x/deployment/install-gateway-apis/ and replaces the install include section with a link to that. This lets us segregate the warnings in a dedicated location. Since you have to pass through that at least once to use GWAPI examples at all, we can keep them out of the guides.

It also separates the instructions for various versions. I've kept the GatewayAlpha section in always with a note saying it's required for 2.6 since I figured that was better than just saying "upgrade to a newer version!" on older pages without noting that you needed additional steps beyond that.